src/plugins/lightscript.js

```diff
@@ -302,6 +302,10 @@ pp.parseArrowType = function (node) {
     case "=*>": case "-*>":
       node.generator = true;
       break;
+    case "=/*>": case "-/*>":
```

Comment:
Sorry, could we do `=*/>` and `-*/>` instead? I prefer it visually and it's less likely to trip up syntax highlighters who think it's a comment. Unless there's a reason you prefer this style?